src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -89,16 +97,18 @@ public ModelManager(ReadOnlyWorkoutPlanner workoutPlanner, ReadOnlyUserPrefs use
         this.userPrefs = new UserPrefs(userPrefs);
         this.workoutPlanner = new WorkoutPlanner(workoutPlanner);
         this.diaryRecords = null;
+        this.dashboards = null;
         filteredPersons = null;
         filteredRecords = null;
         filteredRecipes = new FilteredList<>(this.recipeBook.getRecipeList());
         filteredExercises = null;
         filteredDiaries = null;
+        filteredDashboards = null;
     }
 
     public ModelManager() {
         this(new UserProfile(), new HealthRecords(), new RecipeBook(),
-                new WorkoutPlanner(), new DiaryRecords(), new UserPrefs());
+                new WorkoutPlanner(), new DiaryRecords(), new UserPrefs(), new DashboardRecords());
```

Comment:
For consistency, Dashboard records should be before UserPrefs in the constructor